### PR TITLE
Small code/docs cleanup

### PR DESCRIPTION
**Pull request by JPTIZ**
- Remove unused code
- Slightly enhance and fix a few documentations

### CODE REVIEW

File: paratex/extractor.py [hunk 1]
```diff
@@ -90,6 +93,10 @@ def fetch_sessions_from_interval(
 
 
 def fetch_session(url: str) -> List[Tuple[str, str]]:
+    """
+    Accesses a session's attendance page URL and returns the attencance table
+    columns.
```
Comment (GabrielSimonetto) [inline paratex/extractor.py]:
```suggestion
    Accesses a session's attendance page URL and returns the attendance table
    columns.
```

File: paratex/extractor.py [hunk 2]
```diff
@@ -18,8 +18,7 @@ class Session:
     @staticmethod
     def make_url(session_id: int):
         BASE_URL = (
-            "http://transparencia.alesc.sc.gov.br/"
-            "presenca_plenaria_detalhes.php"
+            "http://transparencia.alesc.sc.gov.br/" "presenca_plenaria_detalhes.php"
```
Comment (GabrielSimonetto) [inline paratex/extractor.py]:
```suggestion
            "http://transparencia.alesc.sc.gov.br/presenca_plenaria_detalhes.php"
```

This max line size on black sounds a bit big no? I preferred the 2 line break here

**Review by GabrielSimonetto — CHANGES_REQUESTED** [review]
Just some typos and stuff

**Comment by JPTIZ** [comment]
These issues should be fixed now.

**Review by GabrielSimonetto — APPROVED** [review]
Nice!

**Pull request merged**